doc/tutorial/Tutorial.tex

```diff
@@ -383,14 +378,14 @@ \section{Introduction to the proof engine: Minimal Logic}
 apply H; [ assumption | apply H0; assumption ].
 \end{coq_example}
 
+It is discouraged though to rely on automatically generated names in finished
```

Comment:
The grammar police say: I think "though" should have commas before and after it. The construct is an "appositive", if I recall.